app/views/studies/_metadata_file_fields.html.erb

```diff
@@ -1,6 +1,46 @@
 <div class="form-group row">
   <div class="col-sm-4">
-    <%= f.label :use_metadata_convention, "Use Metadata Convention?" %> <%= render partial: 'metadata_convention_help_popover', locals: {id: f.object.id.to_s} %><br />
-    <%= f.select :use_metadata_convention, options_for_select([['Yes', true],['No', false]], f.object.use_metadata_convention), {}, class: 'form-control use-metadata-convention', disabled: !f.object.new_record? %>
+    <% convention_required = User.feature_flag_for_instance(current_user, 'convention_required') %>
+    <%= f.label :use_metadata_convention do %>
+      Do you use SCP conventional names for required metadata column headers?
+      <%= render partial: 'metadata_convention_help_popover', locals: {id: f.object.id.to_s} %>
+    <% end %>
+    <br />
+    <%= f.label :use_metadata_convention_true, 'data-analytics-name': 'metadata-convention-optin' do %>
+      <%= f.radio_button :use_metadata_convention, true, checked: true, disabled: !f.object.new_record?%>
+      Yes
+    <% end %>
+    &nbsp;
+    <%= f.label :use_metadata_convention_false, 'data-analytics-name': 'metadata-convention-optout' do %>
+      <% if convention_required %>
+        <a href="#/" style="color:#999" id="convention-decline-label-<%= f.object.id.to_s %>"
+           data-content="Using the convention is now required.  If this presents a particular issue for your study, please contact us at <a href='mailto:scp-support@broadinstitute.zendesk.com'>scp-support@broadinstitute.zendesk.com</a> so we can help or make any updates to the convention."
+           data-toggle="popover"
+           data-analytics-name="convention-decline-label">
+          <%= f.radio_button :use_metadata_convention, false, checked: false, disabled: true %>
+          No
+        </a>
```

Comment:
Out-of-scope, but just to surface recent team chats: 

It's not ideal to show a disabled selection UI that can only be enabled after emailing.  It works, but it's awkward.  It'd likely be better to not show radio buttons in this case, and instead just show an button like "Using the convention is a problem for me" that shows these instructions on click.

Radio buttons make sense if the user is exempted, but not until then.

Reply:
No argument, but this is duplicating the UX from the upload wizard - the main point (other than fixing the bug) was to homogenize the experience so that any time a user is attempting to ingest a metadata file they are presented with the same controls.